recipes/quickfix/all/conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
+  "1.15.1":
+    url: "https://github.com/quickfix/quickfix/archive/76a7b41a40b744fbdcd7a707012f0308b8d4c5ee.tar.gz"
```

Comment:
Why not using https://github.com/quickfix/quickfix/archive/v1.15.1.tar.gz ?

Reply:
Well, that's a good question. Actually this tag is from 2018 - after that the project failed to keep bumping versions. This hash in conandata.yml is from the last commit to the master branch. What do you think is the best way to proceed in this case?

Reply:
Therefore your link is not `1.15.1` :s
Definitly, I would use https://github.com/quickfix/quickfix/archive/v1.15.1.tar.gz . Eventually you could ask to library author for a new release.

Reply:
Ok, this is going to take some time, because the repo changed a bit since then. I'll be working on this at night.

Reply:
Done, it was easier than I thought at first.